docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+    Kind:         CloudsqlInstance
+    Name:         mysqlinstance-6a7fe064-d888-11e9-ab90-42b6bb22213a
+    Namespace:    gcp-infra-dev
+  Write Connection Secret To Ref:
+    Name:  wordpressmysql
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
+Events:                    <none>
+```
+
+*Note: You must wait until the claim becomes bound before continuing with this
+guide. It could take a few minutes for GCP to complete CloudSQL creation.*
+
+#### Connecting MySQL Instance and GKE Cluster
+Before we install Wordpress, we need establish connectivity between our MySQL
+database and our GKE cluster. We can do this by creating a [Private Service 
+Connection](https://cloud.google.com/vpc/docs/configure-private-services-access).
+
+You can create it following the link above or use Crossplane to do it:
+
+* Create a `GlobalAddress` and `Connection` resources:
+```bash
+cat > network.yaml <<EOF
+---
+# example-globaladdress defines the IP range that will be allocated for cloud services connecting
+# to the instances in the given Network.
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: GlobalAddress
+metadata:
+  name: example-globaladdress
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  name: example-globaladdress
+  purpose: VPC_PEERING
+  addressType: INTERNAL
+  prefixLength: 16
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+---
+# example-connection is what allows cloud services to use the allocated GlobalAddress for communication. Behind
+# the scenes, it creates a VPC peering to the network that those service instances actually live.
+apiVersion: servicenetworking.gcp.crossplane.io/v1alpha2
+kind: Connection
+metadata:
+  name: example-connection
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  parent: services/servicenetworking.googleapis.com
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+  reservedPeeringRanges:
+    - example-globaladdress
+EOF
+
+kubectl apply -f network.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl -n $INFRA_NAMESPACE get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
+```
+*Output*
+```
+NAME                 FIRST_CONDITION   SECOND_CONDITION
+example-connection   True              True
+```
+Wait for both conditions to be true to continue.
+
+## Install Wordpress
```

Comment:
Is this part common to all providers? Are we planning on having a single high-level service guide which links out to the provider-specific documents for the provider-specific parts? The benefit would be that we wouldn't need to maintain separate copies of the same thing, and so that the user would have a single high-level guide for the instructions

Reply:
It's not specific to GCP. @prasek will we have a main doc where we refer to cloud-specific guides and link back to the main one to keep going with Wordpress or is it the case that each cloud services guide is standalone guide?